### PR TITLE
Awesome books with Modules project

### PR DESCRIPTION
**Pull request by mandelatuks**
# in this milestone I did the following 

- Use ES6 syntax.
- Divide your code into modules. 
- Keep all functionalities of the app without errors.
- Refactor your methods and functions to arrow functions.
- Use let and const in the correct way.
- Display the current date below the navigation bar using the luxon library

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  bookListSection.innerHTML = awesomeBooks.getBooks().map((book, index) => `
+        <article class="book ${index % 2 === 0 ? 'dark' : ''}">
+            <div>
+                <p class="title">"${book.title}" by ${book.author}</p>
+            </div>
+            <div class="button-remove">
+            <button data-id=${book.id} class="remove">Remove</button>
+            </div>
+        </article>`).join('');
+}
```
Comment (elmar8287) [inline index.js]: - [ ] Please, consider to use ES6 syntax, aka array functions. Here you can use
```
const renderBook = () => {
.....
}
```

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @mandelatuks.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- Nice design 👍 
- No linter issues 💯 
- Informative PR
- Nice commits history (... but for future consider using a capital letter for commit title - last two commits are lowercase) 👍 

## Required Changes ♻️

- [ ] Check the comment under the review.
- [ ] Please, also consider modifying the README file:
       - there is an incorrect screenshot - please, remove it or place the right version
       - fix the live demo version link - it is not working - you can also remove it if the app is not 
       deployed.
       - try to give personality to your MIT.md file - add your name and app name 👍 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (mandelatuks) [reply index.js]: @elmar8287 
thank you so much fixed the issues

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @mandelatuks,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**